XMonad/Hooks/StatusBar/PP.hs

```diff
@@ -101,7 +100,9 @@ data PP = PP { ppCurrent :: WorkspaceId -> String
              , ppWsSep :: String
                -- ^ separator to use between workspace tags
              , ppTitle :: String -> String
-               -- ^ window title format for the focused window
+               -- ^ window title format for the focused window. To
+               -- display the titles from other screens, check
+               -- 'XMonad.Util.Loggers.logTitles'.
```

Comment:
This could perhaps be misinterpreted.  I think the most immediate use-case for `logTitles` is that it also shows the titles of all unfocused windows on the workspace.

Reply:
^^ :+1:

rest of the PR lgtm

Reply:
I really question what I write some times... I'll push a fix

Reply:
I think it's fixed now, right?

Reply:
I went for the second one :+1: